src/test/java/org/folio/rest/impl/EholdingsTitlesTest.java

```diff
@@ -0,0 +1,150 @@
+  private static final String STUB_CUSTOMER_ID = "TEST_CUSTOMER_ID";
+  private static RequestSpecification spec;
+  private static int port;
+  private static String host;
+
+  @org.junit.Rule
+  public WireMockRule userMockServer = new WireMockRule(
+    WireMockConfiguration.wireMockConfig()
+      .dynamicPort()
+      .notifier(new ConsoleNotifier(true)));
+
+  @BeforeClass
+  public static void setUpClass(final TestContext context) {
+    Vertx vertx = Vertx.vertx();
+    vertx.exceptionHandler(context.exceptionHandler());
+    port = NetworkUtils.nextFreePort();
+    host = "http://localhost";
+
+    DeploymentOptions restVerticleDeploymentOptions = new DeploymentOptions()
+      .setConfig(new JsonObject().put("http.port", port));
+    vertx.deployVerticle(RestVerticle.class.getName(), restVerticleDeploymentOptions, context.asyncAssertSuccess());
+
+    spec = new RequestSpecBuilder()
+      .setBaseUri(host + ":" + port)
+      .build();
+  }
+
+  @Before
+  public void setUp() {
+    RMAPIConfigurationCache.getInstance().invalidate();
+  }
+
+  @Test
+  public void shouldReturnTitlesOnGet() throws IOException, URISyntaxException {
+    String stubResponseFile = "responses/rmapi/titles/searchTitles.json";
+
+    String wiremockUrl = host + ":" + userMockServer.port();
+    TestUtil.mockConfiguration("responses/configuration/get-configuration.json", wiremockUrl);
+    WireMock.stubFor(
+      WireMock.get(new UrlPathPattern(new RegexPattern("/rm/rmaccounts/" + STUB_CUSTOMER_ID + "/titles.*"), true))
+        .willReturn(new ResponseDefinitionBuilder()
+          .withBody(TestUtil.readFile(stubResponseFile))));
+
+    RestAssured.given()
+      .spec(spec).port(port)
+      .header(new Header(RestConstants.OKAPI_URL_HEADER, wiremockUrl))
+      .header(TENANT_HEADER)
+      .header(TOKEN_HEADER)
+      .when()
+      .get("eholdings/titles?page=1&filter[name]=Mind&sort=name")
+      .then()
+      .statusCode(200)
+
+      .body("meta.totalResults", equalTo(8766))
+      .body("data[0].type", equalTo("titles"))
+      .body("data[0].id", equalTo("1175655"))
+      .body("data[0].attributes.name", equalTo("The $1 Million Reason to Change Your Mind"))
+      .body("data[0].attributes.publisherName", isEmptyOrNullString())
+      .body("data[0].attributes.isTitleCustom", equalTo(false))
+      .body("data[0].attributes.subjects[0].type", equalTo("BISAC"))
+      .body("data[0].attributes.subjects[0].subject", equalTo("BUSINESS & ECONOMICS / Small Business"))
+
+      .body("data[0].attributes.identifiers[0].id", equalTo("7209484"))
+      .body("data[0].attributes.identifiers[1].id", equalTo("978-1-74216-894-4"))
+      .body("data[0].attributes.identifiers[2].id", equalTo("978-0-7303-7792-4"))
+
+      .body("data[0].attributes.identifiers[0].type", equalTo("ISSN"))
+      .body("data[0].attributes.identifiers[1].type", equalTo("ISBN"))
+      .body("data[0].attributes.identifiers[2].type", equalTo("ISBN"))
+
```

Comment:
Might be helpful to have a comment or even a separate test -- noting that identifiers get sorted by type and subtype. This is happening - here but not apparent from the test